nodes/core/core/80-function.js

```diff
@@ -204,7 +204,12 @@ module.exports = function(RED) {
         }
         var context = vm.createContext(sandbox);
         try {
-            this.script = vm.createScript(functionText);
+            this.script = vm.createScript(functionText, {
+                filename: this.name + '_' + this.id, // filename for stack traces
```

Comment:
I think the virtual filename could be a bit more helpful to the user - remember that `this.name` may be blank.

How about `'Function Node:'+this.id+(this.name?' ['+this.name+']':'')`

which would give: 

```
Error: Error creating a new Tensor2D: values must be number[][] or flat/TypedArray
    at Object.ArrayOps.tensor2d (/Volumes/data/dev/aryballe/aryballe-studio/app/node_modules/@tensorflow/tfjs-core/dist/ops/array_ops.js:56:19)
    at Function Node:da38083d.f04568 [tensorflow]
    at Function Node:da38083d.f04568 [tensorflow]:9:3
```

Reply:
@knolleary great idea. Will change that. Was also thinking about removing the "unwanted" part from the stack (from node-red). What do you think?

Reply:
Yup, if we're adding the stack it should be as useable as possible - doesn't need to be full of internal details

Reply:
Hi @farfromrefug - can we get the filename field updated here and there's a rogue console.log that needs removing. Thanks!

Reply:
@knolleary should be good now